src/styles/PPLoadingSpinner.css

```diff
@@ -1,6 +1,10 @@
 .pp-loading-spinner {
-  width: 40px;
-  height: 40px;
+  width: 80px;
+  height: 80px;
+  position: absolute;
+  margin: auto;
+  top: 0;
+  bottom: 0
```

Comment:
Absolute positioning with auto margins should allow for the spinner to be centered vertically on the whole page view, not just centered vertically within the component itself.